bundles/org.openhab.binding.magentatv/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,258 @@
+				<required>true</required>
+				<options>
+					<option value="Auto">AUTO</option>
+					<option value="DMS_TPB">MR400</option>
+					<option value="MR401B">MR401B</option>
+					<option value="MR201">MR201</option>
+				</options>
+			</parameter>
+			<parameter name="ipAddress" type="text">
+				<label>Receiver IP address</label>
+				<description>IP address of the receiver</description>
+				<required>true</required>
+				<context>network-address</context>
+			</parameter>
+			<parameter name="port" type="text">
+				<label>Receiver Port</label>
+				<description>Port address for UPnP</description>
+				<required>true</required>
+			</parameter>
+			<parameter name="accountName" type="text">
+				<label>Account name</label>
+				<description>Credentials: Login name (xxx@t-online.de)</description>
+				<required>true</required>
+			</parameter>
+			<parameter name="accountPassword" type="text">
+				<label>Account password</label>
+				<description>Credentials: Account password</description>
+				<context>password</context>
+				<required>true</required>
+			</parameter>
+			<parameter name="userId" type="text">
+				<label>User ID</label>
+				<description>Technical User ID for pairing</description>
+				<required>false</required>
+				<advanced>true</advanced>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<channel-group-type id="control">
+		<label>Control</label>
+		<description>Control function for your Media Receiver</description>
+		<channels>
+			<channel id="power" typeId="system.power" />
+			<channel id="player" typeId="system.media-control" />
+			<channel id="stop" typeId="stop" />
+			<channel id="volumeUp" typeId="volumeUp" />
+			<channel id="volumeDown" typeId="volumeDown" />
+			<channel id="mute" typeId="system.mute" />
+			<channel id="channel" typeId="channel" />
+			<channel id="channelUp" typeId="channelUp" />
+			<channel id="channelDown" typeId="channelDown" />
+			<channel id="key" typeId="key" />
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="program">
+		<label>Program Information</label>
+		<description>Information on the running program</description>
+		<channels>
+			<channel id="title" typeId="system.media-title" />
+			<channel id="text" typeId="programText" />
+			<channel id="start" typeId="programStart" />
+			<channel id="duration" typeId="programDuration" />
+			<channel id="position" typeId="programPosition" />
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="status">
+		<label>Play Status</label>
+		<description>Status information on media play</description>
+		<channels>
+			<channel id="channelCode" typeId="channelCode" />
+			<channel id="runStatus" typeId="runStatus" />
+			<channel id="playMode" typeId="playMode" />
+		</channels>
+	</channel-group-type>
+
+	<channel-type id="stop">
+		<item-type>Switch</item-type>
+		<label>Stop</label>
+		<description>Stops the player. ON if the player is stopped.</description>
+	</channel-type>
```

Comment:
doesn't the `player` channel already handle this?

Reply:
channel removed